kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -704,51 +726,76 @@ define([
                 }
                 return counts;
             }, {});
+            /* Second, set up values to only track duplicates, and fold in the
+             * results from all other tabs.
+             *
+             * values should now look like:
+             * {
+             *   'field_value': {
+             *     otherTabs: {
+             *        tabId: [rows],
+             *     },
+             *     thisTab: {
+             *        [{ rowId1, paramId }, { rowId2, paramId }]
+             *     }
+             *   }
+             * }
+             * with only entries where there are at least two row ids, or
+             * a value found from another tab
+             */
             const values = Object.keys(outputValueCounts).reduce((duplicateValues, value) => {
-                if (outputValueCounts[value].length > 1) {
-                    duplicateValues[value] = outputValueCounts[value];
+                const dup = {};
+                if (outputValueCounts[value].length > 1 || value in otherTabOutputValues) {
+                    dup.thisTab = outputValueCounts[value];
+                }
+                if (value in otherTabOutputValues) {
+                    dup.otherTabs = otherTabOutputValues[value];
+                }
+                if (Object.keys(dup).length) {
+                    duplicateValues[value] = dup;
                 }
                 return duplicateValues;
             }, {});
-            // values should now look like:
-            // {
-            //   'field_value': [{ rowId1, paramId }, { rowId2, paramId }]
-            // }
-            // with only entries where there are at least two row ids
-            // translate to something easier to work with -
-            // {
-            //    rowId: {
-            //      paramId1: {
-            //        widget: Object
-            //        duplicateRows: [rowId, rowId]
-            //      },
-            //      paramId2: {
-            //        widget: Object,
-            //        duplicateRows: [rowId, rowId]
-            //    }
-            // }
+            /* Get the parameter widgets and translate to something easier to work with -
+             * {
+             *    rowId: {
+             *      paramId1: {
+             *        widget: Object
+             *        duplicateRows: {
+             *          thisTab: [rowId, rowId],
+             *          otherTabs: {
+             *              tabId1: [rowIdx, rowIdx]
```

Comment:
is it deliberate that `thisTab` uses `rowId` and `otherTabs` uses `rowIdx`?

Reply:
Yeah, that's a typo. At that point, they should all numerical row index values, not the unique ids.